pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaManaged.hpp

```diff
@@ -419,6 +443,23 @@ class StateVectorCudaManaged
             const std::vector<std::string> names(tgtsInt.size(), "I");
             applyParametricPauliGeneralGate_(names, ctrlsInt, ctrls_valuesInt,
                                              tgtsInt, 2 * params[0], adjoint);
+        } else if (opName == "PCPhase") {
+
```

Comment:
Do we want to remove the blank lines at the start and the end of each codeblock?